cli/command/inspect/inspector.go

```diff
@@ -7,10 +7,11 @@ import (
 	"strings"
 	"text/template"
 
-	"github.com/docker/cli/cli"
-	"github.com/docker/cli/templates"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/templates"
```

Comment:
Also looks to be accidentally changed (we don't use 3 groups currently in this repo, only two)